linux/scripts/reconf.sh

```diff
@@ -43,20 +33,17 @@ fi
 
 echo "Found tier ${TIER}, version ${VERSION}"
 
-# autoreconf the projects
-for proj in ${autotool_projects}; do
-    cd "$proj"
-    echo "Reconfiguring $proj to version ${VERSION}"
-    autoreconf -if
-    rm -rf autom4te.cache
-    cd "$BASEDIR"
-done
-
 for proj in ${extra_projects}; do
     if [ "${proj}" == "keyboardprocessor" ] || [ "${proj}" == "keyman" ]; then
         rm -rf keyboardprocessor
         cp ../VERSION.md ../core/
         ../core/build.sh -t keyboardprocessor configure
+        # we need to build core first before we can configure ibus-keyman
+        ../core/build.sh -t keyboardprocessor build
```

Comment:
Does this mean that we could build keyboardprocessor twice? Looking above it seems that `keyboardprocessor` and `keyman` never end up specified together, so perhaps this is okay, but it seems kinda funny.